CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 3.0.0
+
+* Heavy architectural changes to make the plugin more modular
+* Added support for Bitbucket.org repositories (Mercurial and Git)
+* Added support for GitLab repositories
+* Added support for self-managed repositories (GitLab and others)
+* Added new REST API route for incoming webhooks ( `traduttore/v1/incoming-webhook`)
```

Comment:
```suggestion
* Added new REST API route for incoming webhooks (`traduttore/v1/incoming-webhook`) which replaces the deprecated (`github-webhook/v1/push-event`)
```

Reply:
Shouldn't this be "… the deprecated _one_ (…)"?

Reply:
Yes, there was something missing. I updated it.